packages/ui/test/membership/hooks/useMembers.test.tsx

```diff
@@ -0,0 +1,104 @@
+import { renderHook } from '@testing-library/react-hooks'
+
+import { MembershipOrderByInput } from '@/common/api/queries'
+import { MemberListFilter, MemberListEmptyFilter } from '@/memberships/components/MemberListFilters'
+import { MemberListOrder, DefaultMemberListOrder, useMembers } from '@/memberships/hooks/useMembers'
+import { asMember, Member } from '@/memberships/types'
+import { MockMember, mockMembers } from '@/mocks/data/mockMembers'
+
+import { useFilterMembersQuery } from '../../../src/memberships/queries'
+
+const { EntryAsc, EntryDesc, HandleAsc } = MembershipOrderByInput
+
+interface RenderUseMembers {
+  filter?: Partial<MemberListFilter>
+  order?: Partial<MemberListOrder>
+}
+const renderUseMembers = ({ filter = {}, order = {} }: RenderUseMembers) =>
+  renderHook(() =>
+    useMembers({
+      filter: { ...MemberListEmptyFilter, ...filter },
+      order: { ...DefaultMemberListOrder, ...order },
+    })
+  )
+
+jest.mock('../../../src/memberships/queries', () => ({
+  useFilterMembersQuery: jest.fn(() => ({})),
+}))
+const mockedUseFilterMembersQuery = useFilterMembersQuery as jest.Mock
+
+describe('useMyMember', () => {
+  beforeEach(() => {
+    mockedUseFilterMembersQuery.mockClear()
+  })
+
+  it('calls the useFilterMembersQuery', () => {
+    renderUseMembers({})
+    expect(useFilterMembersQuery).toBeCalled()
+  })
+
+  it('queries members by ascending ids (default)', () => {
```

Comment:
Hey, we follow a different naming for tests. You can try to change the names to follow that, but that's not a requirement.

In this case, we would name the test: "Default order". (Starting with a capital letter).

The other test names could be:

*   Inverse order
*   Order by handle
*   Search by handle
*   Search by Id

This is to:

*   have the description of the test be more compact
*   describe the outcome/state rather than what the test does

It differs from the more broadly used "sentence test description". However, I find it better in terms of readability when a test fails.

Reply:
Sounds good. I prefer to have the test suite fitting with the others